### DISCUSSION TITLE
How to send multiple files at once?

### DISCUSSION THREAD

**Discussion by mat-twg** [yagop/node-telegram-bot-api#1176] [General]
Hi! Is it possible to send multiple files like `sendMediaGroup` at one message?

**Comment by Zoomish** [comment]
Hi! You can use sendMediaGroup to send multiple files in one message:

```javascript
bot.sendMediaGroup(chatId, [
    { type: 'photo', media: 'https://example.com/img1.jpg' },
    { type: 'photo', media: 'https://example.com/img2.jpg' },
    { type: 'document', media: fs.createReadStream('file.pdf') }
]);
```

Note:

· Supports up to 10 files
· File types: photo, video, document, audio
· Only the first item can have a caption

Hope that helps! Let me know if you need more details.
